### ISSUE TITLE
Priority adjustment for encoder servers

### ISSUE THREAD

**Issue by JoshWho** [WWBN/AVideo#7706]
I would like my encoders to be used first before the Encoder Network using the public encoders,
 but for some reason the encoder network always chooses your public servers first. 

Any way I can make it use my available encoders first?

![download (19)](https://user-images.githubusercontent.com/39392449/220831470-fa977f05-e575-4c3f-a52b-1205b09e9af6.png)

**Comment by DanielnetoDotCom** [comment]
not really, it just checks who is available or who has the best network connection/CPU consumption

**Comment by JoshWho** [comment]
Any way to make it just which one is not in use? I kind of think at this point with networks are all mostly high speed now but still is a floating variable not needed anymore.

The reason why I ask was it kept defaulting to your public one and it wasn't letting users import from youtube. Not sure if it is fixed now or not.
